src/app/auth/services/session-storage.service.ts

```diff
@@ -0,0 +1,30 @@
+import {Inject, Injectable} from '@angular/core';
+import {DOCUMENT} from "@angular/common";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SessionStorageService {
+
+  SESSION_TOKEN: string = 'session-token';
+
+  private window: Window;
+
+  constructor(@Inject(DOCUMENT) private document: Document) {
+    // @ts-ignore
+    this.window = this.document.defaultView;
+  }
+
+  public setToken(token: string) {
+    window.sessionStorage.setItem(this.SESSION_TOKEN, token)
```

Comment:
```suggestion
    this.window.sessionStorage.setItem(this.SESSION_TOKEN, token)
```
Without `this` means that we use global `window` object, not class field.